README.md

```diff
@@ -5,86 +5,113 @@
+
+```
+sudo microceph cluster bootstrap
+```
+
+> [!NOTE]  
+> `cluster` is a microceph subcommand for managing clusters.
+
+After setup, add storage to your cluster with:
+
+```
+sudo microceph disk add loop,4G,3
+```
+
+Here, you’ll add three virtual disks (“loop file” disks) of 4 GiB each.
+
+
+Once your cluster is set up and running, you can monitor its status with:
+
+```
+sudo microceph status
+```
+
+Note that there are no spaces between the `disk add` arguments.
 
-## 💡 Philosophy
 
-Deploying and operating a Ceph cluster is complex because Ceph is designed to
-be a general-purpose storage solution. This is a significant overhead for small
-Ceph clusters. MicroCeph solves this by being _opinionated_ and _focused_ on
-the small scale. With MicroCeph, deploying and operating a Ceph cluster is as
-easy as a [Snap][snap-microceph]!
+If you need a comprehensive status report of your cluster, including its health and disk usage, run:
 
-## 🎯 Features
+```
+sudo microceph.ceph status
+```
 
-* Quick and consistent deployment with minimal overhead
-* Single-command operations (for bootstrapping, adding OSDs, service enablement, etc)
-* Isolated from the host and upgrade-friendly
-* Built-in clustering so you don't have to worry about it!
+## Documentation
 
-## ⚡️ Quickstart
+The [MicroCeph documentation][rtd-microceph] contains guides and learning material about
+what you can do with MicroCeph and how it works.
 
-The below commands will set you up with a testing environment on a single
-machine using file-backed OSDs - you'll need about 15 GiB of available space on
-your root drive:
+Documentation is maintained in the [`docs`][docs-dir-microceph] directory of this repository.
+It is written in reStructuredTest (reST) format, built with Sphinx, and published on Read The Docs. 
 
-    sudo snap install microceph 
-    sudo snap refresh --hold microceph
-    sudo microceph cluster bootstrap
-    sudo microceph disk add loop,4G,3
+## Project and Community
 
-You're done! Check Ceph status:
+MicroCeph is a member of the Ubuntu family. It's an open-source project that warmly welcomes community contributions,
+suggestions, fixes, and constructive feedback.
 
-    sudo ceph status
+If you find any errors or have suggestions for improvements, please [open an issue on GitHub][bug-microceph]
 
-You can remove everything cleanly with:
+[Join our Matrix forum][matrix-microceph] to engage with our community and get support.
 
-    sudo snap remove microceph --purge
+We abide by the [Ubuntu Code of Conduct][ubuntu-coc].
 
-## 📖 Documentation
+Excited about MicroCeph? If you star the project on GitHub, you'll become a [Stargazer][stargazers-microceph]!
 
-The documentation is found in the [`docs`][docs-dir-microceph] directory. It is
-written in RST format, built with Sphinx, and published on Read The Docs:
+## Contribute to MicroCeph
 
-[MicroCeph documentation][rtd-microceph]
+MicroCeph is growing as a project, and we would love your help.
```

Comment:
how about `MicroCeph is rapidly growing, and we would love your help.` ?

Reply:
It is growing rapidly, isn't it? :)